src/d3d11/d3d11_blend.cpp

```diff
@@ -235,5 +314,56 @@ namespace dxvk {
       Logger::err(str::format("D3D11: Invalid logic op: ", LogicOp));
     return VK_LOGIC_OP_NO_OP;
   }
+
+  bool D3D11BlendState::ValidateBlend(const D3D11_BLEND blend) {
+    return blend >= D3D11_BLEND_ZERO
+        && blend <= D3D11_BLEND_INV_SRC1_ALPHA;
+  }
+
+  bool D3D11BlendState::ValidateBlendAlpha(const D3D11_BLEND blendAlpha) {
+    //can't be color operations
```

Comment:
Instead of blacklist, with weird ranges outside possible enums, wouldn't whitelist be better?

Reply:
Well, in this situation,  a whitelist would have 13 comparisons versus 8 comparisons on the blacklist.